templates/100.html

```diff
@@ -5,7 +5,7 @@
         <h1>100 theorems</h1>
 
         <p><a href="https://www.cs.ru.nl/~freek/">Freek Wiedijk</a> maintains <a href="https://www.cs.ru.nl/~freek/100/">a list</a> tracking progress of theorem provers in formalizing 100 classic theorems in mathematics as a way of comparing prominent theorem provers.
-        Currently {{hundred_theorems|selectattr('formalized')|list|length}} of them are formalized in Lean.
+        Currently {{hundred_theorems|selectattr('proof_formalized')|list|length}} of them are formalized in Lean.
```

Comment:
Can you copy the text from `1000.html` and `1000-missing.html` about formalized statements to this page and `100-missing.html`?

Reply:
Done!